---
title: About
layout: page
---
![Profile Image]({{ site.url }}/{{ site.picture }})

<p>Dr Camelia Quek is a NHMRC Investigator Fellow at Melanoma Institute Australia, appointed as an Adjunct Associate Professor at the University of Sydney, and serve as a Steering Committee at the Cancer Research Network. Dr Quek originally did a first degree in molecular biology at the University of New South Wales (University Medal in Molecular Biology) before moving on to do a PhD in genomics and bioinformatics at the University of Melbourne (Sawyer Medal). Dr Quek has a diploma in Molecular Biotechnology. Dr Quek is an international expert in the integration and interpretation of high-dimensional data obtained from cancer patients, and uses these discoveries to address most pressing questions in the field. Dr Quek’s early postdoctoral research has made significant advances in understanding cancer biology and tumour microenvironment in melanoma patients treated with immunotherapy, using gene expression profiles to discover biomarkers in response to cancer therapies, and single-cell multi-omics to guide clinical decision-making.</p>
<p>Her research impact and contributions are recognised in Australia and worldwide. Over the past years, she has received several prestigious awards including 2022 CINSW Premier’s Awards for Wildfire Highly Cited Publication (co-first authors, Cancer Cell 2019), 2021 10x Millennium Science Start Single Cell Award, and 2017 CINSW Premier’s Awards for Excellence in Translational Research Team. She has also received 3x Best Poster Winner 2014, 2017 and 2018, and 2x Best Oral Talk 2017 and 2023 at both international and national conferences.</p>
<p>Dr Quek’s research program focuses on understanding the immunogenic expression profiles in patients treated with cancer therapies, key proteins and biological mechanisms driving cellular identity and heterogeneity within tumour and immune microenvironment, particularly in the context of biomarker, drug resistance, druggable targets and how tumour heterogeneity shapes disease progression. Her current research interests are: (1) Biomarkers of response and resistance to cancer therapies, (2) Single-cell multi-omics to understand treatment and progression, and (3) Spatial imaging to dissect key cellular interactions and novel drug targets.</p>
<p>Aside from research, Dr Quek is appointed as an Associate Editor in Molecular Carcinogenesis (2023 - Present), Review Editor at Frontiers (2021 - Present), Special Issue Editor (2021 - Present) in International Journal of Molecular Sciences (IJMS), and as well as Genes. She is also a frequent reviewer for Nature, Cell Reports Medicine, BMC Cancer, IJMS, Frontiers in Oncology, and Frontiers in Digital Health. In addition, Dr Quek is an Ambassador (2017 - Present) for European Association of Cancer Research. For teaching, she is a PhD thesis examiner, and a leading supervisor for several Honours and PhD students, and chairs PhD progress evaluation meeting. Furthermore, Dr Quek is a co-investigator in Data Access Committee, where sequencing data generated for the purposes of biomedical, clinical and translational research projects are frequently deposited and shared in TCGA, ENA and EGA so that researchers can undertake similar studies worldwide.</p>


<h2>Skills</h2>

<ul class="skill-list">
	<li>Project management</li>
	<li>Problem solving</li>
	<li>Stakeholders and communications</li>
	<li>Programming such as R, Python, Shell scripting</li>
	<li>HTML/CSS</li>
	<li>Next-generation sequencing library design (10x Genomics, Illumina, 454 and Ion Torrent)</li>
	<li>Immunohistochemistry</li>
	<li>Biochemistry, proteomic, genetic, cell and molecular biology techniques</li>
	<li>Bioinformatics and big data analysis including DNA-/RNA-seq, single-cell (scRNAseq, CITEseq, TCRseq, CyTOF), muultiplex immunohistochemistry (7-plex or Phenocycler), spatial biology, gene expression profiling, epitope prediction, network/pathway, phylogenetic and virology</li>
</ul>

<h2>Experience</h2>

<ul>

<b>Cancer:</b>
<p>My current research interests are: (1) Biomarkers of response and resistance to cancer therapies, (2) Single-cell multi-omics to understand treatment and progression, (3) Spatial imaging to dissect tumour microenvironment and key cellular interactions, and (4) novel drug discovery.</p>
   	<li><a href="https://aacrjournals.org/clincancerres/article/29/13/2513/727344/Intratumoral-CD16-Macrophages-Are-Associated-with">Intratumoral CD16+ macrophages are associated with clinical outcomes of patients with metastatic melanoma treated with combination anti-PD-1 and anti-CTLA-4 therapy</a></li>
	<li><a href="https://jitc.bmj.com/content/11/10/e007144.info">Classification of the tumour immune microenvironment and associations with outcomes in patients with metastatic melanoma treated with immunotherapies</a></li>	
   	<li><a href="https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-023-04092-9">Cross-platform comparison of immune signatures in immunotherapy-treated patients with advanced melanoma using a rank-based scoring approach</a></li>
    	<li><a href="https://www.mdpi.com/2073-4425/12/10/1629">High-Dimensional Single-Cell Transcriptomics in Melanoma and Cancer Immunotherapy</a></li>
      	<li><a href="https://insight.jci.org/articles/view/157839">Distinct pretreatment innate immune landscape and posttreatment T cell responses underlie immunotherapy-induced colitis</a></li>
	<li><a href="https://aacrjournals.org/clincancerres/article/29/1/154/711979/Obesity-Is-Associated-with-Altered-Tumor">Obesity is associated with altered tumor metabolism in metastatic melanoma</a></li>
 	<li><a href="https://www.mdpi.com/2073-4425/13/10/1757">Unravelling Tumour Microenvironment in Melanoma at Single-Cell Level and Challenges to Checkpoint Immunotherapy</a></li>
  	<li><a href="https://www.mdpi.com/2073-4425/12/10/1629">High-Dimensional Single-Cell Transcriptomics in Melanoma and Cancer Immunotherapy</a></li>
   	<li><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00885-z">Cryopreservation of human cancers conserves tumour heterogeneity for single-cell multi-omics analysis</a></li>
    	<li><a href="https://acsjournals.onlinelibrary.wiley.com/doi/full/10.1002/cncr.32522">Site-specific response patterns, pseudoprogression, and acquired resistance in patients with melanoma treated with ipilimumab combined with anti-PD-1 therapy</a></li>
  	<li><a href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(19)30037-6">Anti-PD-1 Monotherapy and Anti-PD-1/Anti-CTLA-4 Combined Therapy</a></li>
        	<li><a href=" ">Recurrent hotspot SF3B1 mutations at codon 625 in vulvovaginal mucosal melanoma identified in a study of 27 Australian mucosal melanomas</a></li>
      	<li><a href="https://aacrjournals.org/clincancerres/article/26/2/487/82689/Macrophage-Derived-CXCL9-and-CXCL10-Are-Required">Macrophage-Derived CXCL9 and CXCL10 Are Required for Antitumor Immune Responses Following Immune Checkpoint Blockade</a></li>
       	<li><a href="https://aacrjournals.org/clincancerres/article/25/11/3247/81607/Prevalence-and-Cellular-Distribution-of-Novel">Prevalence and Cellular Distribution of Novel Immune Checkpoint Targets Across Longitudinal Specimens in Treatment-naive Melanoma Patients: Implications for Clinical Trials</a></li>
		<li><a href="https://www.tandfonline.com/doi/full/10.1080/2162402X.2019.1659093">Close proximity of immune and tumor cells underlies response to anti-PD-1 based therapies in metastatic melanoma patients</a></li>
  	<li><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ijc.31791">Whole genome sequencing of melanomas in adolescent and young adults reveals distinct mutation landscapes and the potential role of germline variants in disease susceptibility</a></li>
   	<li><a href="https://www.tandfonline.com/doi/full/10.1080/2162402X.2018.1537581">Integrated molecular and immunophenotypic analysis of NK cells in anti-PD-1 treated metastatic melanoma patients</a></li>
    	<li><a href="https://aacrjournals.org/clincancerres/article/25/4/1272/10155/Distinct-Molecular-Profiles-and-Immunotherapy">Distinct Molecular Profiles and Immunotherapy Treatment Outcomes of V600E and V600K BRAF-Mutant Melanoma</a></li>
     	<li><a href="https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2018.00584/full">The Prognostic Significance of Low-Frequency Somatic Mutations in Metastatic Cutaneous Melanoma</a></li>
      	<li><a href="https://aacrjournals.org/clincancerres/article/24/13/3036/80949/CD103-Tumor-Resident-CD8-T-Cells-Are-Associated">CD103+ Tumor-Resident CD8+ T Cells Are Associated with Improved Survival in Immunotherapy-Naïve Melanoma Patients and Expand Significantly During Anti–PD-1 Treatment</a></li>

<br><br>

<b>Neurodegenerative diseases:</b>
<p>My PhD research has bridged wet-lab based biochemistry and molecular biology
with computational biology in identifying driver genes and the RNA content of small extracellular vesicles known as exosomes, with the objective of restructuring the approach of disease diagnosis and treatment. I am also an expert in data analytics especially in studies with longitudinal samples. I developed a new pipeline, <a href="http://israp.sourceforge.net/">iSRAP</a>, for the high-throughput sequence analysis of extracellular vesicle RNA that has enabled researchers worldwide to undertake similar studies.</p>
	<li><a href="https://www.nature.com/articles/s42003-021-01868-x">Distribution of microRNA profiles in pre-clinical and clinical forms of murine and human prion disease</a></li>
 	<li><a href="https://www.tandfonline.com/doi/full/10.1080/15476286.2016.1270005">Defining the purity of exosomes required for diagnostic profiling of small RNA suitable for biomarker discovery</a></li>
  	<li><a href="https://www.sciencedirect.com/science/article/pii/S0006291X16315571?via%3Dihub">The role of extracellular vesicles in neurodegenerative diseases</a></li>
    	<li><a href="https://www.nature.com/articles/srep22398">Gene dysregulation is restored in the Parkinson’s disease MPTP neurotoxic mice model upon treatment of the therapeutic drug CuII(atsm)</a></li>
      	<li><a href="https://www.frontiersin.org/articles/10.3389/fgene.2013.00150/full">The detection of microRNA associated with Alzheimer's disease in biological fluids using next-generation sequencing technologies</a></li>

<br><br>

<b>Immunology:</b>
<p>As part of a work collaboration, I have contributed the data analytics and my knowledge of Immunology (from Onco-immunology, Neuro-immunology and Infectious diseases).</p>
	<li><a href="https://www.mdpi.com/1422-0067/24/11/9334">Non-Antigenic Modulation of Antigen Receptor (TCR) Loop Modulates Signalling: Implications of External Factors Influencing T-Cell Responses</a></li>

<br><br>

<b>Virology:</b>
<p>My early research including Honours focuses on virology. I lead the statistical or computational biology component of the work.</p>
	<li><a href="https://www.mdpi.com/2075-4418/13/8/1364)">Multiplex RT Real-Time PCR Based on Target Failure to Detect and Identify Different Variants of SARS-CoV-2: A Feasible Method That Can Be Applied in Clinical Laboratories.</a></li>
 	<li><a href="https://www.mdpi.com/2075-4418/12/10/2393">Characteristics of Hepatitis B Virus Genotype and Sub-Genotype in Hepatocellular Cancer Patients in Vietnam</a></li>
 	<li><a href="https://www.sciencedirect.com/science/article/pii/S1525157817302969?via%3Dihub">Evaluation of a Hepatitis C Virus Core Antigen Assay in Plasma and Dried Blood Spot Samples</a></li>

<br><br>

 <b>Software and analysis tools:</b>
 <p>I developed or contributed to software and analysis tools as part of my work.</p>
	<li><a href="https://github.com/cameliaquek/singlecell_spatial_multiomics/">MIT</a></li>
 	<li><a href="http://israp.sourceforge.net/">iSRAP</a></li>
	<li><a href="http://www.funrich.org/">FunRich</a></li>

<br><br>

  <b>Large datasets:</b>
  <p>I manage or contribute to large or big datasets that received high international impact from data sharing of high-value pre-clinical or clinical trial samples.</p> 
 	<li><a href="https://ega-archive.org/studies/EGAS00001006977">EGAS00001006977: NanoString nCounter® PanCancer IO 360™on anti-PD1/anti-PD1+CTLA4 in patients with metastatic melanoma
</a></li>
 	<li><a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB54666">PRJEB54666: The 16s rRNA sequencing data from OpACIN-neo clinical trial.
</a></li>
 	<li><a href="https://ega-archive.org/studies/EGAS00001006982">EGAS00001006982: Metagenomics shotgun sequencing was conducted on fecal samples from the Australian patients enrolled in the OpACIN-neo clinical trial</a></li>
 	<li><a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB45779">Clinical and Molecular Heterogeneity in Patients with Innate Resistance to Anti-PD-1 +/− Anti-CTLA-4</a></li>
	<li><a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB23709">Biomarkers of response and resistance to checkpoint blockade immunotherapy in metastatic melanoma</a></li>
 	<li><a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB6957">Whole transcriptome profiling of the MPTP neurotoxic mice model reveals potential targets to modulate gene expression</a></li>
</ul>
